web/src/user_topic_popover.ts

```diff
@@ -28,24 +32,30 @@ export function initialize() {
             popover_menus.popover_instances.change_visibility_policy = instance;
             popover_menus.on_show_prep(instance);
             const $elt = $(instance.reference).closest(".change_visibility_policy").expectOne();
-            const stream_id = $elt.attr("data-stream-id");
-            const topic_name = $elt.attr("data-topic-name");
+            const stream_id_str = $elt.attr("data-stream-id");
+            const topic_name_str = $elt.attr("data-topic-name");
```

Comment:
Is the `topic_name_str` variable serving any purpose other than to  just be assigned to a more useful `topic_name` variable four lines downstream? If not than probably, we can just
```
topic_name = $elt.attr("data-topic-name")!;
```